### PR TITLE
Accept json schema

### LINKED ISSUE #36: Accept JSON schema in init function
In order to support https://github.com/Zooz/express-ajv-swagger-validation/issues/80, we need to allow this behavior in api-schema-builder

### PR DESCRIPTION
**Pull request by kobik**
Closes #36

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,31 +25,43 @@ function buildSchema(swaggerPath, options) {
...
+function buildSchemaSync(pathOrSchema, options) {
+    const jsonSchema = getJsonSchema(pathOrSchema);
+    const dereferencedSchema = deref(jsonSchema);
 
-    return buildValidations(parsed, dereferenced, options);
+    return buildValidations(jsonSchema, dereferencedSchema, options);
+}
+
+function getJsonSchema(pathOrSchema) {
+    if (pathOrSchema instanceof Object) {
```
Comment (kibertoad) [inline src/index.js]: Isn't it easier to check if `pathOrSchema` is string? Judging by Lodash implementation, check is not that simple and there might be edge cases (https://github.com/lodash/lodash/blob/master/isObject.js)

File: test/openapi3/general/general-oai3-test.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-let chai = require('chai').use(require('chai-as-promised')),
-    expect = chai.expect,
-    schemaValidatorGenerator = require('../../../src/index'),
-    path = require('path');
+const chai = require('chai').use(require('chai-as-promised'));
+const path = require('path');
+const yaml = require('js-yaml');
+const fs = require('fs');
+
+const schemaValidatorGenerator = require('../../../src/index');
```
Comment (kibertoad) [inline test/openapi3/general/general-oai3-test.js]: isn't `index` at the end redundant?

Comment (kobik) [reply test/openapi3/general/general-oai3-test.js]: yep

File: src/index.js (hunk 1 above)
Comment (kobik) [reply src/index.js]: fixed
